src/Concordium/Client/Output.hs

```diff
@@ -675,6 +675,8 @@ showEvent verbose = \case
     in Just $ printf "Transfer memo:\n%s" str
   Types.Interrupted cAddr _ ->
     verboseOrNothing [i|interrupted '#{cAddr}'.|]
+  Types.Upgraded{..} ->
+    verboseOrNothing [i|upgraded contract instance at '#{euAddress}' from #{euFrom} to #{euTo}.|]
```

Comment:
```suggestion
    verboseOrNothing [i|upgraded contract instance at '#{euAddress}' from '#{euFrom}' to '#{euTo}'.|]
```

Reply:
Seems like we quote the module references elsewhere.

Reply:
Quoted now.